### DISCUSSION TITLE
Andel tillæg

### DISCUSSION THREAD

**Discussion by Munar18** [rndfm/elspotpris#88] [Ideas]
Hej, jeg er meget imponeret over jeres arbejde. Bruger siden dagligt, da jeg arbejder med el. 

Jeg har en opdatering til Andels Variabel el, der er på lagt 0,14 kr. som system tillæg.

**Comment by rndfm** [comment]
Er det et yderligere tillæg til spotprisen ud over det vi har registreret i dag eller det totale tillæg?

**Reply by rndfm** [reply]
Gælder det også for deres "nye" TimeEnergi?

**Comment by rndfm** [comment]
Jeg går ud fra der er tale om en misforståelse. Systemtariff mv til energinet løber op i 13,75 øre inkl moms.

Hvis det ikke er tilfældet og det rent faktisk er et Andel Energi tillæg så må du meget gerne kontakte mig.

**Comment by Munar18** [comment]
Det var en fejl, med den gamle pris på basis el. Men deres time energi, som
er deres eneste variable pris, er betydeligt dyre pr. KWh time i gennemsnit

man. 19. dec. 2022 kl. 15.46 skrev Christian Reinholdt <
***@***.***>:
